modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -2861,9 +2861,7 @@ cancelled=Canceled
 cancelling=Canceling
 cannot-be-added-because-a-required-x-has-been-deleted=Cannot be added because a required {0} has been deleted.
 cannot-be-found={0} cannot be found.
-cannot-copy-file-entries-to-the-live-version-of-a-group=Cannot copy file entries to the live version of a group. Make your changes in staging and publish them to Live afterwards to make them public.
-cannot-copy-file-shortcuts-to-the-live-version-of-a-group=Cannot copy file shortcuts to the live version of a group. Make your changes in staging and publish them to Live afterwards to make them public.
-cannot-copy-folders-to-the-live-version-of-a-group=Cannot copy folders to the live version of a group. Make your changes in staging and publish them to Live afterwards to make them public.
+cannot-copy-to-the-live-version-of-a-group=Cannot copy to the live version of a group. Make your changes in staging and publish them to Live afterwards to make them public.
```

Comment:
I think this message should say "site" instead of "group", as @AliciaGarciaGarcia already pointed out, see https://github.com/liferay-lima/liferay-portal/pull/4702#discussion_r1330935296